<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SensorCfgDlg</class>
 <widget class="QDialog" name="SensorCfgDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sensor Configuration</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Send Sensor measurements</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbPublishTo">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>291</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbFastCadence">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbCadenceInOut">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbTriggerType">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>210</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Send data</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>150</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>the range</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Publish sensor data every</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>110</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>times more often</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>86</x>
     <y>150</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>if measurements are</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Publish data if measurement increases by</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>250</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>seconds</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>60</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>msec</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConfigPub">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>156</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Configure publication</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConfigCadence">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>250</y>
     <width>156</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Configure cadence</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chbPubPeriod">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Publish periodically</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chbFastCadence">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Fast cadence</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chbTriggers">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>92</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Triggers</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edSendInterval">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>250</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edPubFreq">
   <property name="geometry">
    <rect>
     <x>462</x>
     <y>60</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edCadenceHigh">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>150</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtriggerUp">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>210</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edTriggerDown">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>210</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edCadenceLow">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>150</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>150</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>210</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>or decreases by</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Do not send more often than</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
